code/modules/mob/living/living_recoil.dm

```diff
@@ -1,64 +1,36 @@
-/mob/living/proc/handle_recoil(var/obj/item/gun/G)
+/mob/living/proc/handle_recoil(var/obj/item/gun/G, var/recoil_buildup)
 	deltimer(recoil_reduction_timer)
-	if(G.one_hand_penalty) // If the gun has a two handed penalty and is not wielded.
-		if(!G.wielded)
-			recoil += G.one_hand_penalty // Then the one hand penalty will be added to the recoil.
-
-	if(G.braced)
-		if(G.braceable > 1)
-			recoil *= 0.5 // Taking penalty into account, bipod reduces recoil by 40%.
-	else if(G.brace_penalty)
-		recoil += G.brace_penalty
 
 	var/debug_recoil = min(0.3, G.fire_delay)
 	if(G.fire_delay == 0)
 		debug_recoil = 0.3
 	if(G.burst > 1)
 		debug_recoil = max(debug_recoil, G.burst_delay)
 
-	add_recoil(G.recoil_buildup + debug_recoil) // DEBUG, remove the debug_recoil after testing is over!
+	add_recoil(recoil_buildup + debug_recoil) // DEBUG, remove the debug_recoil after testing is over!
```

Comment:
![image](https://user-images.githubusercontent.com/65828539/162937506-62d87759-9cda-415a-8809-5bd75f3e6c95.png)

Reply:
Thx for the reminder, I'll have to get around to this too